src/app.js

```diff
@@ -81,134 +33,59 @@ var run = function() {
 			}
 			window.localStorage[key] = content;
 		}
-		SOL_CACHE_FILE = fileKey(Object.keys(files)[0]);
+		editor.setCacheFile(Object.keys(files)[0]);
```

Comment:
That function should be named differently, but can / should be done in a subsequent refactoring step.